### DISCUSSION TITLE
The problem with running spigot server.

### DISCUSSION THREAD

**Discussion by hamx01** [itzg/docker-minecraft-server#1652] [Q&A]
Hi! I just discovered, that in docker you can run mc-server.
I have already done spigot 1.19 server, with plugins and DynMap, and it's working if you run it without docker.
But how can I run it in docker container?
I tried this command: docker run -it -d -v /minecraft-server:/data 
-e TYPE=SPIGOT 
-e BUILD_FROM_SOURCE=true
-p 25565:25565 
-e EULA=TRUE 
--name mc itzg/minecraft-server
But, it just crashes container.
I have these files: 
![obraz](https://user-images.githubusercontent.com/101278038/183218206-504cd7ee-7305-4706-a41e-fafafd71a6f5.png)
How can I run this jar file? Inside docker container, of course.

**Comment by itzg** [comment]
Providing the container logs will be necessary for anyone to help. 

Also, why are you choosing to build from source?

**Reply by hamx01** [reply]
![obraz](https://user-images.githubusercontent.com/101278038/183241895-80bdfd84-a05e-496e-a17c-e329a76346f2.png)
From logs, I see that I have an older version. Can I define version while running container?

**Reply by itzg** [reply]
That's fair then. It does look like they're a few days behind

https://getbukkit.org/download/spigot
